packages/api/cms-api/src/dam/images/images.service.ts

```diff
@@ -22,14 +22,11 @@ export class ImagesService {
         return createHmac("sha1", this.config.secret).update([fileHash, cropHash].join(":")).digest("hex");
     }
 
-    createUrlTemplate(
-        { file, cropArea }: ImageInterface,
-        { previewDamUrls = false, relativeDamUrls = false }: { previewDamUrls?: boolean; relativeDamUrls?: boolean },
-    ): string {
+    createUrlTemplate({ file, cropArea }: ImageInterface, { previewDamUrls = false }: { previewDamUrls?: boolean }): string {
         const imageCropArea = cropArea !== undefined ? cropArea : file.image!.cropArea;
         const filename = parse(file.name).name;
 
-        const baseUrl = [`${relativeDamUrls ? "" : this.config.apiUrl}/dam/images`];
```

Comment:
you maybe can remove apiUrl from config now(?)

Reply:
8eac6a2

Reply:
is it still needed in the config?